playbooks/collect_instance_statistics/collect_elasticsearch_data.yml

```diff
@@ -0,0 +1,42 @@
+---
+# A simple one-off playbook that gathers basic usage information from OpenedX hosts.
+# This is used by the `activity_csv` management command.
+
+- hosts: all
+  vars:
+    # This should be overridden to be more unique.
+    local_output_dir: /tmp/
+    remote_output_filename: /tmp/activity_report
+  tasks:
+    # We have to do a copy+command pair here (instead of script) in order to get
+    #   separate channels for stdout and stderr.
+    # See also: https://github.com/ansible/ansible-modules-core/pull/4000/files
+    - name: Copy query_elasticsearch script
+      copy:
+        src: query_elasticsearch.py
+        dest: /tmp/query_elasticsearch.py
+      become: yes
+      become_user: www-data
+
+    - name: Run query_elasticsearch script
+      command: >
+        /edx/bin/python.edxapp /tmp/query_elasticsearch.py {{ server_name_prefix }} {{ num_days }} {{ remote_output_filename }}
+      register: stats_results
+      become: yes
+      become_user: www-data
```

Comment:
@pcockwell, again, no `/edx/bin/python.edxapp`. You will have to dynamically create a virtualenv environment, install the requirements, run the script and clean it up even if there are errors in the previous step(s). See https://docs.ansible.com/ansible/2.5/user_guide/playbooks_blocks.html#error-handling